lib/evaluatable.rb

```diff
@@ -0,0 +1,67 @@
+module Evaluatable
+  # This function take 3 args and check
+
+  # - if they are equal, return true
+
+  # - if thery are not equal, return false
+
+  def are_equal?(the_a, the_b, the_c, player, opponent)
+    the_a == the_b && the_b == th_c && the_c == the_b && (the_a == player || the_a == opponent)
+  end
+
+  def player?(board_cell, player, opponent)
+    return 10 if board_cell == player
+
+    return -10 if board_cell == opponent
+  end
+
+  # This function take a board a evaluate it
+
+  # - if player wom, return 10
+
+  # - if opponent wom, return -10
+
+  # - if there is no winer, return 0
+
+  def evaluate(board, player = 'o', opponent = 'x')
+    # Checking rows for victory
+
+    3.times do |row|
+      if are_equal?(board[row][0], board[row][1], board[row][2], player, opponent)
+
+        return is_player?(board[row][0], player, opponent)
+
+      end
+    end
+
+    # Checking cols for victory
+
+    3.times do |col|
+      if are_equal?(board[0][col], board[1][col], board[2][col], player, opponent)
+
+        return is_player?(board[0][col], player, opponent)
+
+      end
+    end
+
+    # Checking frist diagonal for victory
+
+    if are_equal?(board[0][0], board[1][1], board[2][2], player, opponent)
+
+      return is_player?(board[0][0], player, opponent)
+
+    end
+
+    # Checking second diagonal for victory
+
+    if are_equal?(board[0][2], board[1][1], board[2][0], player, opponent)
+
+      return is_player?(board[0][2], player, opponent)
+
+    end
```

Comment:
- I'll advise you amend your loop structure to this :point_down: 
- You should also use the variable below to terminate your variable
```
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```